Source/Core/VideoCommon/Assets/DirectFilesystemAssetLibrary.cpp

```diff
@@ -47,50 +48,47 @@ DirectFilesystemAssetLibrary::GetLastAssetWriteTime(const AssetID& asset_id) con
 CustomAssetLibrary::LoadInfo DirectFilesystemAssetLibrary::LoadTexture(const AssetID& asset_id,
                                                                        CustomTextureData* data)
 {
-  if (auto iter = m_assetid_to_asset_map_path.find(asset_id);
-      iter != m_assetid_to_asset_map_path.end())
-  {
-    const auto& asset_map_path = iter->second;
+  const auto asset_map = GetAssetMapForID(asset_id);
```

Comment:
I chose to prefer a copy instead of holding onto the lock for the whole time that the texture was loaded